src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/BuildingGeometryContext.cs

```diff
@@ -25,6 +25,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .Property(x => x.BuildingPersistentLocalId)
                 .HasColumnName("PersistentLocalId");
 
+            modelBuilder.Entity<BuildingGeometryData>()
+                .Property(x => x.StatusAsString).HasColumnName("Status");
```

Comment:
@ArneD , I didn't use HasConversion for the Status property as we want to filter on it. This is correct reasoning?

Reply:
Since it's only used for filtering, it's good!